app/repositories/AnalyticsDataCache.scala

```diff
@@ -10,7 +10,15 @@ import services.Dynamo
 import scala.util.control.NonFatal
 import scala.collection.JavaConversions._
 
-abstract sealed trait CacheResult[+A] extends Product
+abstract sealed trait CacheResult[+A] extends Product {
```

Comment:
`Option` is abstract because it isn't a trait: http://www.scala-lang.org/api/current/scala/Option.html